util/helpers.py

```diff
@@ -224,6 +224,35 @@ def getLogger(name, logLvl=logging.INFO):
     return l
 
 
+def getLogLevel(logLvl):
+    """
+    From a string (by argparse) that could be INT or STR, check and solve the LogLevel.
+    If not possible or bad option, return logging.NOTSET | 0.
+
+    Return: logging.LogLvl
+    """
+    logNameToLevel = {
+        "CRITICAL": logging.CRITICAL,
+        "FATAL": logging.CRITICAL,
+        "ERROR": logging.ERROR,
+        "WARN": logging.WARNING,
+        "WARNING": logging.WARNING,
+        "INFO": logging.INFO,
+        "DEBUG": logging.DEBUG,
+        "NOTSET": logging.NOTSET,
+    }
+
+    try:
+        lvl = int(logLvl)
```

Comment:
Sounds good assuming that only developers will be using this feature. If was something for normal users, i would prefer another not so hard way to break/resolve the error.

Reply:
yes, i think this will be right like this:

`lvl = int(logLvl)
assert (
    logging.NOTSET <= lvl <= logging.CRITICAL
), "level out of scope: {} < lvl < {}".format(logging.NOTSET, logging.CRITICAL)
return lvl`

Reply:
Sounds right

Reply:
My suggestions was bad, as per https://github.com/decred/tinydecred/pull/47#discussion_r364914910. Instead, do 

```
if lvl > logging.CRITICAL or lvl < logging.NOTSET: 
    raise AssertionError("...")
```